resource-managers/mesos/src/test/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterSchedulerSuite.scala

```diff
@@ -278,6 +278,31 @@ class MesosClusterSchedulerSuite extends SparkFunSuite with LocalSparkContext wi
     assert(uris.asScala.forall(_.getCache))
   }
 
+  test("SPARK-26082 supports setting fetcher cache in the submission") {
```

Comment:
```
- test("SPARK-26082 supports setting fetcher cache in the submission") {
+ test("SPARK-26192 supports setting fetcher cache in the submission") {
```

Reply:
Ah, right - actually, can I just get rid of the JIRA tag?

Reply:
For this case, yep. If you want, you can remove it.

Reply:
Done